velox/dwio/dwrf/test/TestReader.cpp

```diff
@@ -90,7 +90,8 @@ void verifyFlatMapReading(
   rowReaderOpts.select(std::make_shared<ColumnSelector>(requestedType));
   auto reader =
       DwrfReader::create(std::make_unique<FileInputStream>(file), readerOpts);
-  auto rowReader = reader->createRowReader(rowReaderOpts);
+  auto rowReaderOwner = reader->createRowReader(rowReaderOpts);
+  auto rowReader = dynamic_cast<DwrfRowReader*>(rowReaderOwner.get());
```

Comment:
nit: use `static_cast`. No point in using `dynamic_cast` if we are not checking the result for nullptr.

Reply:
I'd prefer to use `dynamic_cast` to have `nullptr` in case of a wrong cast with the following `SEGFAULT` rather than an incorrectly casted pointer.